accession-commons-test/src/main/java/uk/ac/ebi/ampt2d/test/testers/AccessioningServiceTester.java

```diff
@@ -94,7 +98,16 @@ public AccessionWrapperCollectionTester get(List<TestModel> models) {
     public AccessionWrapperCollectionTester getAccessions(String... accessionIds) {
         return addToCollection(singleVersionResults,
                 new AccessionWrapperCollectionTester(() ->
-                        accessioningService.getByAccessions(Arrays.asList(accessionIds))));
+                        getLatestVersionOfAccessions(accessionIds)));
+    }
+
+    private List<AccessionWrapper<TestModel, String, String>> getLatestVersionOfAccessions(String[] accessionIds)
+            throws AccessionDoesNotExistException, AccessionMergedException, AccessionDeprecatedException {
```

Comment:
I think you need to catch exceptions and skip those problematic accessions.

Reply:
no right now we gonna retrieve single accessions only, so it should throw exception if its not in the system.